src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,42 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    public void size() {
+        Integer setSize = numbers.size();
+        Assertions.assertThat(setSize).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(setSize).isEqualTo(3);
```
assertThat 메서드는 테스트에 자주 사용하고, 용도가 명확해서 static import를 사용하는 것을 추천드립니다 :)